ReSharper.FSharp/src/FSharp/FSharp.Psi/src/Impl/Tree/PrimaryConstructorDeclaration.cs

```diff
@@ -13,6 +13,8 @@ public override TreeTextRange GetNameRange() =>
     protected override IDeclaredElement CreateDeclaredElement() =>
       new FSharpPrimaryConstructor(this);
 
+    public ITokenNode EqualsToken => null;
```

Comment:
What do you think about moving the following here, from `collectTypeHintAnchorsForConstructor`?
```fsharp
let owner = FSharpTypeDeclarationNavigator.GetByPrimaryConstructorDeclaration(ctor)
isNotNull owner && isNotNull owner.EqualsToken
```

We could then have a doc comment saying something like ``Returns associated `=` token, possibly from containing type declaration``.